charts/trento-server/Chart.yaml

```diff
@@ -7,7 +7,7 @@ description: The trento server chart contains all the components necessary to ru
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates
-version: 2.3.0
+version: 2.3.1-dev1
```

Comment:
We need not to bump this, right?

Reply:
true, as we didn't change the chart itself it shouldn't be necessary anymore